plugins/genesys/README.md

```diff
@@ -28,27 +28,22 @@ This plugin supports the following protocol IDs:
 
 These devices use the standard USB DeviceInstanceId values for the USB Hub, e.g.
 
-* GenesysLogic USB2.1 Hub: `USB\VID_05E3&PID_0610`
-* GenesysLogic USB2.1 Hub: `USB\VID_05E3&PID_0630`
-* HP USB-C Controller: `USB\VID_03F0&PID_0610`
+* `USB\VID_05E3&PID_0610` (quirk-only)
 
-These devices use the standard USB DeviceInstanceId values for the HID under USB Hub, e.g.
+These devices use the standard USB DeviceInstanceId values for the HID under USB hub, e.g.
 
-* GenesysLogic USB HID device: `USB\VID_05E3&PID_0102`
-* GenesysLogic USB HID device: `USB\VID_05E3&PID_0155`
+* `USB\VID_05E3&PID_0102` (quirk-only)
 
 Additionally, some customized instance IDs are added. e.g.
 
-* GenesysLogic USB2.1 Hub: `USB\VID_03F0&PID_0610&IC_352330&BONDING_0F`
-* GenesysLogic USB2.1 Hub: `USB\VID_03F0&PID_0610&VENDOR_GENESYSLOGIC&IC_352330&BONDING_0F&PORTNUM_23&VENDORSUP_C09B5DD3-1A23-51D2-995A-F7366AAB3CA4`
-* GenesysLogic USB2.1 Hub: `USB\VID_05E3&PID_0630&PROJECT_1885D34D-0418-5EF8-8E69-4CEF77B6B6E8`
-* HP M24fd USB-C Hub: `USB\VID_03F0&PID_0610&PUBKEY_AB859399-95B8-5817-B521-9AD8CC7F5BD6`
-* HP M27fd USB-C Hub: `USB\VID_03F0&PID_0610&PUBKEY_6BE97D77-C2BA-5AA2-B7DF-B9B318BEC2B5`
+* `USB\VID_03F0&PID_0610&IC_352330&BONDING_0F`
+* `USB\VID_03F0&PID_0610&VENDOR_GENESYSLOGIC&IC_352330&BONDING_0F&PORTNUM_23&VENDORSUP_C09B5DD3-1A23-51D2-995A-F7366AAB3CA4`
+* `USB\VID_05E3&PID_0630&PROJECT_1885D34D-0418-5EF8-8E69-4CEF77B6B6E8`
+* `USB\VID_03F0&PID_0610&PUBKEY_AB859399-95B8-5817-B521-9AD8CC7F5BD6`
 
 These devices also use custom GUID values for the Scaler, e.g.
 
-* HP M24fd USB-C Monitor: `GENESYS_SCALER\MSTAR_TSUM_G&PUBKEY_B335BDCE-7073-5D0E-9BD3-9B69C1A6899F&PANELREV_RIM101`
-* HP M27fd USB-C Monitor: `GENESYS_SCALER\MSTAR_TSUM_G&PUBKEY_847A3650-8648-586B-83C8-8B53714F37E3&PANELREV_RIM101`
+* `GENESYS_SCALER\MSTAR_TSUM_G&PUBKEY_B335BDCE-7073-5D0E-9BD3-9B69C1A6899F&PANELREV_RIM101`
```

Comment:
Why only one monitor left here?

Reply:
It's supposed to be an example of structure -- not an enumerated list.